### DISCUSSION TITLE
Random motion artifacts for MRI?

### DISCUSSION THREAD

**Discussion by Can-Zhao** [Project-MONAI/model-zoo#563] [General]
I was wondering if we are interested in more MRI data augmentation like:
https://torchio.readthedocs.io/transforms/augmentation.html#torchio.transforms.RandomMotion
https://torchio.readthedocs.io/transforms/augmentation.html#torchio.transforms.RandomGhosting

**Comment by ericspod** [comment]
I think we would like to have more of these and/or integrate more of TorchIO into MONAI. I had in the past implemented some simple MR augmentations like [this](https://github.com/Project-MONAI/model-zoo/blob/bce5c6695c98351e22b10c43b85a89445b80f9ac/models/valve_landmarks/scripts/valve_landmarks.py#L247) but haven't gotten into committing these to core. If there's others like what you've mentioned that people would like to implement we can discuss and create some interest. Could you please raise an issue on Core regarding this? If there's other issues already open please reference them as well. Thanks!

**Reply by Can-Zhao** [reply]
Sure. Thank you! I've moved it to issues
